specification/edge/resource-manager/Microsoft.Edge/sites/readme.md

```diff
@@ -24,6 +24,12 @@ directive:
   - suppress: OperationsAPIImplementation
     from: sites.json
     reason: RP is in PublicPreview and no SDK has been released yet. Microsoft.Edge RP consist of multiple resources which are owned/maintained by different teams, so we follow folder structure for Service Group (explained here https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster?tab=readme-ov-file#folder-structure-for-service-group). We do have operations api exposed from common-location/folder (https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/edge/resource-manager/Microsoft.Edge/edge/preview/2024-02-01-preview/operations.json#L46C5-L46C43) so every resource need not expose it separately. There has been open issue [Avocado] Support service group folder scenario azure-sdk-tools#6201 for the same.
+  - suppress: TenantLevelAPIsNotAllowed
+    from: sites.json
+    reason: Adding tenant level API to enable site creation on service groups, have got the case reviewed from ARM and PAS team as per documentation
+  - suppress: AvoidAdditionalProperties
+    from: sites.json
+    reason: labels describe user defined tags to be used on Sites.
 ```
```

Comment:
pl add a where clause to the suppressions, so its clear what specific instance of the violation you are requesting the suppression for..

Reply:
Added where clause

Reply:
Approved suppressions.